WebRoot/public/login.jsp

```diff
@@ -256,7 +256,20 @@
                         
                         <c:set var="isUpgradedMailbox" value="${isUpgradedMailbox}" />
                         <c:set var="prefClientType" value="${requestScope.authResult.prefs.zimbraPrefClientType[0]}" />
-                        
+
+                        <c:if test="${param.screenSize eq 'small' && empty client}">
```

Comment:
@k-kato 
What happen if user logged in the classic client and then reduce the screen size and reload the browser.
Will it switch to modern client ?

Reply:
@miteshsavani810 It depends on conditions.

(A) When zimbraPrefClientType is advanced and Default client is selected in login page, `client=advanced` is not added to URL parameter. In this case, when screen size is reduced and the page is reloaded, it is redirected to `https://host/?screenSize=small` to access login page. If Modern UI is supported (`/modern` exists or the browser is not IE11), it is redirected to Modern client. If not, the user logs out and an error is shown in login page.

(B) When Classic client is selected in login page, `client=advanced` is added to URL parameter regardless of zimbraPrefClientType. In this case, when screen size is reduced and the page is reloaded, it is not redirected. Classic UI is still shown. It is processed by `<c:if test="${param.client ne 'advanced'}">` in launchZCS.jsp at line 330.

Reply:
This means, the redirection does not happen when user selects Classic in login page explicitly.
Then the error message includes `If you still want to use it, please choose "Classic" in Web App Version.` as defined in https://github.com/Zimbra/zm-ajax/pull/121.

Reply:
Ok. Got it. Thanks.